src/components/Footer/FooterExtendedNavList/FooterExtendedNavList.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState, useEffect } from 'react'
+import classnames from 'classnames'
+import { NavList } from '../../header/NavList/NavList'
+
+export type ExtendedNavLinksType = React.ReactNode[][]
+
+type FooterExtendedNavListProps = {
+  /* 
+    Multidimensional array of grouped nav links. Sub-arrays are column sections, first element is used as a heading.
+  */
+  nestedLinks: ExtendedNavLinksType
+}
+
+export const FooterExtendedNavList = ({
+  className,
+  nestedLinks,
+}: FooterExtendedNavListProps &
+  React.HTMLAttributes<HTMLElement>): React.ReactElement => {
+  const classes = classnames('grid-row grid-gap-4', className)
+
+  const [isMobile, setIsMobile] = React.useState(false)
```

Comment:
This is super minor but I noticed there is a brief flash of the desktop nav at mobile sizes on render. I think that could be resolved by doing the initial window size check here to default the state to the correct value (instead of `false`) and then that would remove the need to trigger `handleResize()` in the effect hook